src/components/LanguageOption.tsx

```diff
@@ -146,8 +146,22 @@ export default function LanguageOption(props: LanguageOptionProps) {
       .then(() => {
         const metadataExisted = Boolean(metadata?._createdAt)
 
-        callback?.(newTranslationDocument, newMetadataDocument, client)
-
+        try {
+          callback?.({
```

Comment:
Sorry for the mess around on the props here, but it occurred to me that maybe the source document should be available here. It's very plausible that we might want to do something with this document. I'd suggest the following props:

- sourceDocument
- newDocument
- sourceLanguageId
- destinationLanguageId
- metaDocumentId
- client

I _think_ that should be OK!

Reply:
Great stuff!